styles/Stars.module.css

```diff
@@ -0,0 +1,12 @@
+.hidden {
+  display: none;
+}
+
+.star {
```

Comment:
could we also add an outline around the stars

Reply:
I tried a few variations of this. Unfortunately the way we render the stars we can't get an outline or shadow on the star itself, only on the box around the star. This could be resolved with clip-path, or totally re-doing how we render the stars...

I do agree though the styling needs some work. Not sure which direction to go... Borders would match the radio button style, maybe editing the svg is the way to go?

You have any other ideas

Reply:
Opened https://github.com/ibi-group/survey/issues/18